app/helpers/application_helper.rb

```diff
@@ -1,5 +1,7 @@
 module ApplicationHelper
   include Pagy::Frontend
+  include Turbo::FramesHelper
+  include Turbo::StreamsHelper
   def full_title page_title
     base_title = t("base_title")
```

Comment:
Bỏ ngoặc chỗ này có bị lỗi k á huynh. Huynh check lại giúp bạn với
```suggestion
    base_title = t "base_title"
```

Reply:
k lỗi đâu á
sửa r nha